tests/v2/test_waku_filter.nim

```diff
@@ -11,7 +11,7 @@ import
   libp2p/multistream,
   libp2p/transports/transport,
   libp2p/transports/tcptransport,
-  ../../waku/protocol/v2/[waku_filter, filter],
+  ../../waku/protocol/v2/[waku_filter, message_notifier],
```

Comment:
does this belong in protocol?

Reply:
If we put it in node, we get weird import cycles I believe. Not sure tho